docs/troubleshoot/trouble.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Troubleshooting
+nav_order: 10
+has_children: true
+permalink: /troubleshooting
+---
+
+# Troubleshooting
```

Comment:
Wondering if this should go as a subsection in the respective sections like - Minimal downtime etc. Maybe we can only have a FAQ here and trouble shooting in the respective context. But I do not have a strong opinion, so you can decide to ignore.